concent-builder/configure.yml

```diff
@@ -58,9 +75,34 @@
             state:  present
             groups: docker
 
+        - name:    Add jenkins user to sudo group
+          user:
+            name:   jenkins
+            state:  present
+            groups: sudo
+
         - name:    Create data_dir
           file:
             path:  "{{ data_dir }}"
             state: directory
             owner: "{{ shared_user }}"
             group: "{{ shared_user }}"
+
+        - name:    Create jenkins home directory
+          file:
+            path:  "{{ jenkins_home_dir }}/"
+            state: directory
+            owner: jenkins
+            group: jenkins
+
+        - name:   Add jenkins role to sudoers
+          copy:
+            src:  "jenkins-configuration-files/jenkins_sudoers"
+            dest: "/etc/sudoers.d/jenkins"
+
+        - name:   Add inventory file to run ansible playbook by jenkins
+          copy:
```

Comment:
I think you should use `ansible_inventory` and just add `--connection=local` to the commandline where needed.

Reply:
Ok. I found the way to do this by ansible plugin.

Reply:
Why do you need a plugin for this? It works for me without any plugins.